src/components/AdminSettings.vue

```diff
@@ -500,7 +500,12 @@ export default {
 		userAppPasswordHint() {
 			const linkText = t('integration_openproject', 'Administration > File storages')
 			const htmlLink = `<a class="link" href="${this.adminFileStorageHref}" target="_blank" title="${linkText}">${linkText}</a>`
-			return t('integration_openproject', 'This value is only visible now and then never again. Copy this password to OpenProject {htmlLink} as an Administrator.', { htmlLink }, null, { escape: false, sanitize: false })
+			return t('integration_openproject', 'This value will only be accessible once. Now, as an administrator copy this password to OpenProject {htmlLink}.', { htmlLink }, null, { escape: false, sanitize: false })
+		},
+		errorHintForGroupFolderNameAlreadyExists() {
+			const linkText = t('integration_openproject', 'troubleshooting guide')
+			const htmlLink = `<a class="link" href="/404" target="_blank" title="${linkText}">${linkText}</a>`
```

Comment:
```suggestion
			const htmlLink = `<a class="link" href="https://www.openproject.org/docs/system-admin-guide/integrations/nextcloud/" target="_blank" title="${linkText}">${linkText}</a>`
```

I think we can use this link here, the help guide will probably be added here

Reply:
sure